### ISSUE TITLE
Create your own blocking page

### ISSUE THREAD

**Issue by fabionar** [proginosko/LeechBlock#21]
"file:///C:/website.html"

I already tried with so many different html, and none are working. It used to work fine.

**Comment by proginosko** [comment]
Yeah, it looks like Firefox 57 has clamped down security restrictions and won't allow extensions to open local files.

https://discourse.mozilla.org/t/opening-links-to-local-files-file/16449/7

Hard to know how to get around this restriction.

**Comment by proginosko** [comment]
One workaround would be to host the page on a free(ish) hosting site (e.g., Google Sites).

**Comment by matkoniecz** [comment]
> (e.g., Google Sites).

Or github pages.
